spec/integration/kibana/kb_entry/spec.yml

```diff
@@ -0,0 +1,27 @@
+spec:
+  additionalContents: false
+  totalContentsLimit: 50
+  contents:
+    - description: Folder containing a single knowledge base entry definition
+      type: folder
+      pattern: '^[a-z0-9_]+[a-z0-9]$'
+      required: true
+      additionalContents: false
+      contents:
+        - description: A knowledge base entry's manifest file
+          type: file
+          contentMediaType: "application/x-yaml"
+          sizeLimit: 2MB
+          name: "manifest.yml"
+          required: true
+          $ref: "./manifest.spec.yml"
+        - description: A knowledge base entry's index mapping
+          type: file
+          contentMediaType: "application/json"
+          sizeLimit: 5MB
+          name: "index-mapping.json"
+          required: true
+        - description: A knowledge base entry's content file
+          type: file
+          contentMediaType: "application/json"
+          pattern: '^content(\.[0-9]+)?\.json$'
```

Comment:
As suggested by @jsoriano, we're allowing an arbitrary number of file following the fixed `content.json` and 
 dynamic `content.{num}.json` naming format. Each file have a single `documents` property containing the source of the documents we will be indexing into the KB's index.

I did not add any validation on the file's content format, both because I wasn't sure we can add validation on JSON documents, and also because we wouldn't have much to validation (only thing we could do it make sure the only top level property present is `documents` and that it's an array).

But if there is a way to perform validation on JSON docs with the spec validator, I can absolutely add it.

Reply:
> In other cases where we allow multiple files we allow to use letters too, so it can give meaningful names when this is used for organizational purposes, and we use to separate with hyphens.

Sounds good, I will follow the pattern for consistency. I'll just wait for the other question about the format (json vs ndjson) to be addressed to perform both changes at the same time if required.